pkg/operator/ceph/cluster/cluster.go

```diff
@@ -283,7 +283,8 @@ func (c *cluster) notifyChildControllerOfUpgrade() error {
 			cephFilesystem.Labels = map[string]string{}
 		}
 		cephFilesystem.Labels["ceph_version"] = version
-		_, err := c.context.RookClientset.CephV1().CephFilesystems(c.Namespace).Update(&cephFilesystem)
+		cephFileSystem := cephFilesystem
```

Comment:
Let's establish a naming convention so it will be more obvious why we're creating the local variable. Instead of changing the case, what about the prefix `local` for the vars?
```suggestion
		localCephFilesystem := cephFilesystem
```

Reply:
Done.